### PR TITLE
Invitemembers

### PR DESCRIPTION
**Pull request by naveenpaul**

### CODE REVIEW

File: webapp/app/common/context/sessionContext.tsx [hunk 1]
```diff
@@ -7,39 +7,30 @@ import eventService from "@/app/services/eventService";
...
     authService
       .whoAmI()
       .then((res: AxiosResponse<LoggedInUserDto>) => {
         setUser(res.data);
-        if (!isOss) {
-          eventService.send({
-            event: "login",
-          });
-        }
+        eventService.send({
```
Comment (aruntcet2010) [inline webapp/app/common/context/sessionContext.tsx:27]: @naveenpaul @franklinio is it okay to remove this?

File: webapp/app/common/utils/buttonHandler.ts [hunk 2]
```diff
@@ -6,18 +6,12 @@ export interface ButtonMeta {
...
-) {
+function buttonHandler(buttonMeta: ButtonMeta, onClick?: () => void) {
   return () => {
-    if (!isOss) {
-      eventService.send({
-        event: buttonMeta.id + "_clicked",
-        ...buttonMeta.dataLayer,
-      });
-    }
+    eventService.send({
```
Comment (aruntcet2010) [inline webapp/app/common/utils/buttonHandler.ts:11]: same here

**Review by aruntcet2010 — CHANGES_REQUESTED** [review]

File: webapp/app/common/context/sessionContext.tsx (hunk 1 above)
Comment (franklinio) [reply webapp/app/common/context/sessionContext.tsx:27]: @aruntcet2010 I think an old main was merged back to this branch
@naveenpaul Please pull and rebase your changes to the head of the main.

File: webapp/app/common/utils/buttonHandler.ts (hunk 2 above)
Comment (naveenpaul) [reply webapp/app/common/utils/buttonHandler.ts:11]: @aruntcet2010 we don't need these events to be captured?

**Pull request closed without merging**